targettests/execution/state_init_err_runtime.cpp

```diff
@@ -9,8 +9,6 @@
 // clang-format off
 // Note: change |& to 2>&1| if running in bash
 // RUN: nvq++ %cpp_std %s -o %t --target quantinuum --emulate && %t |& FileCheck %s
-// Note: change |& to 2>&1| if running in bash
-// RUN: nvq++ %cpp_std --enable-mlir --target remote-mqpu --remote-mqpu-url localhost:9999 %s -o %t && %t |& FileCheck %s
```

Comment:
If this test is being removed, do we need to update the wording for the error message on line 34 to remove the reference to remote simulators not being able to synthesize it anymore?

In other words, perhaps the error should be updated to say `op synthesis: unsupported argument type for remote devices: state*`

Reply:
yes, thanks!

Reply:
fixed:)